api/src/services/accesscontrols.js

```diff
@@ -1,5 +1,7 @@
 const AccessControl = require('accesscontrol');
 
```

Comment:
`datasetUploads` are not exposed as a resource anymore. Instead, the `dataset` resource is exposed directly. This is in accordance with the redesign of the API to use the existing `/datasets` route for API uploads, instead of having a `/datasetUploads` route.